Source/WebCore/dom/ElementRareData.h

```diff
@@ -61,6 +61,10 @@ class ElementRareData : public NodeRareData {
     explicit ElementRareData();
     ~ElementRareData();
 
+    enum class RareDataFlags : uint8_t {
```

Comment:
We have more bits available in EventTargetFlag. Why not add a flag there instead of creating an entire ElementRareData?